src/components/templates/Jazzbar/JazzBarSkeletonPage/JazzBarSkeletonPage.tsx

```diff
@@ -20,9 +21,11 @@ const JazzBarSkeletonPage: React.FunctionComponent<PropsType> = ({
 }) => {
   const venue = useSelector(currentVenueSelector);
 
+  const infoIcon = venue?.host?.icon || getDefaultPortalIcon(venue?.template);
```

Comment:
And yes, if this looks complicated, it can be encapsulated into a function named `determinePortalIcon()` or similar.

```suggestion
  const infoIcon =
    venue?.host?.icon ||
    (SPACE_PORTALS_ICONS_MAPPING[venue?.template ?? ""] ?? DEFAULT_VENUE_LOGO);
```

Reply:
https://github.com/sparkletown/sparkle/pull/2606#discussion_r758985324